src/MAUI/Maui.Samples/Samples/Layers/AddVectorTiledLayerFromCustomStyle/AddVectorTiledLayerFromCustomStyle.xaml.cs

```diff
@@ -0,0 +1,165 @@
+﻿// Copyright 2023 Esri.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at: http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific
+// language governing permissions and limitations under the License.
+
+using ArcGIS.Samples.Managers;
+using Esri.ArcGISRuntime.Mapping;
+using Esri.ArcGISRuntime.Portal;
+using Esri.ArcGISRuntime.Tasks.Offline;
+
+namespace ArcGIS.Samples.AddVectorTiledLayerFromCustomStyle
+{
+    [ArcGIS.Samples.Shared.Attributes.Sample(
+        name: "Add vector tiled layer from custom style",
+        category: "Layers",
+        description: "Load ArcGIS vector tiled layers using custom styles.",
+        instructions: "Pan and zoom to explore the vector tile basemap.",
+        tags: new[] { "tiles", "vector", "vector basemap", "vector tiled layer", "vector tiles" })]
+    [ArcGIS.Samples.Shared.Attributes.OfflineData("f4b742a57af344988b02227e2824ca5f")]
+    public partial class AddVectorTiledLayerFromCustomStyle
+    {
+        // ArcGIS Online portal item strings.
+        private readonly string[] _onlineItemIDs =
+        {
+            "1349bfa0ed08485d8a92c442a3850b06",
+            "bd8ac41667014d98b933e97713ba8377",
+            "02f85ec376084c508b9c8e5a311724fa",
+            "1bf0cc4a4380468fbbff107e100f65a5",
+        };
+
+        private readonly string[] _offlineItemIDs =
+        {
+            "e01262ef2a4f4d91897d9bbd3a9b1075",
+            "ce8a34e5d4ca4fa193a097511daa8855"
+        };
+
+        // Items to fill the ActionDisplaySheet for switching custom styles.
+        private readonly string[] styles = { "Default", "Style 1", "Style 2", "Style 3", "Offline custom style: Light", "Offline custom style: Dark" };
```

Comment:
```suggestion
        private readonly string[] styles = { "Default", "Style 1", "Style 2", "Style 3", "Offline custom style - Light", "Offline custom style - Dark" };
```

Feel free to ignore but I think changing the style names here to fit the UI suggestion might be good.